bin/terminus

```diff
@@ -12,22 +12,21 @@
 // Unset memory limit
 ini_set('memory_limit', -1);
 
-// Version compatibility check and messaging
 if (version_compare(PHP_VERSION, '7.4.0', '<') === true) {
     echo "\n";
     echo 'Sorry, your PHP version (' . PHP_VERSION . ') is no longer supported.' . "\n";
     echo 'Upgrade to PHP 7.4 or newer to use Terminus 3. For PHP versions prior to 7.4, downgrade to Terminus 2.x.'; echo "\n\n";
     exit(1);
-} elseif (version_compare(PHP_VERSION, '8.1.0', '>=') === true) {
+}
+
+if (!getenv('TERMINUS_ALLOW_UNSUPPORTED_NEWER_PHP') && version_compare(PHP_VERSION, '8.2.0', '>=') === true) {
```

Comment:
The check for `getenv('TERMINUS_ALLOW_UNSUPPORTED_NEWER_PHP')` needs to be at the top since any `echo` would bloat commands' responses.